test/system/src/test/java/io/pravega/test/system/AbstractScaleTests.java

```diff
@@ -37,10 +37,10 @@
     private final ConnectionFactory connectionFactory = new ConnectionFactoryImpl(false);
     @Getter(lazy = true)
     private final ClientFactory clientFactory = new ClientFactoryImpl(SCOPE, new ControllerImpl(getControllerURI(),
-            ControllerImplConfig.builder().retryAttempts(1).build(), getConnectionFactory().getInternalExecutor()));
```

Comment:
please also make this change in all the system tests

Reply:
done